### PR TITLE
[Planting Flow] Input Forms Styling Fix

### PR DESCRIPTION
**Pull request by hmzakhalid**
## Pull Request Checklist

- [x] I have read the CONTRIBUTING document.

## Type
Enhancement

## Body
This PR is a fix to #465 assigned to me. 

## Changes made

Created a new Theme for Planting Drawer Form for consistency. 
Font Size increased and Margin between fields added. 
The selected field gets highlighted with green color. 

## Images
Increased Font Size
image

Highlight selected
image

This is my first open source PR, so please any feedback is Highly appreciated. Thanks!

### CODE REVIEW

**Comment by ri0nardo** [comment]
@hmzakhalid this looks good. I like the spacing you picked. Question on the scientific name, does the dropdown menu of species still appear?

**Comment by hmzakhalid** [comment]
> @hmzakhalid this looks good. I like the spacing you picked. Question on the scientific name, does the dropdown menu of species still appear?

Yup, Everything else works as before
image

File: client/src/components/App/App.css [hunk 1]
```diff
@@ -29,7 +29,9 @@
   --font-family-monospace: SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace;
 }
 
-*, *::before, *::after {
```
Comment (mwpark2014) [inline client/src/components/App/App.css:32]: Is there anything to call out in this file? I believe these are all auto-formatting changes

File: client/src/components/Form/FormTheme.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+          fontSize: fontSize,
+        },
+      },
+    },
+
+    MuiFormControl: {
+      styleOverrides: {
+        root: {
+          marginBottom: '1.2rem',
+          label: {
```
Comment (mwpark2014) [inline client/src/components/Form/FormTheme.js:32]: nit: I see some repeatable elements here. 

Potentially can store in a variable and use in multiple component style overrides
```
const rootBaseStyle = 
{
  label: {
    fontSize: labelFontSize,
  },
  '& label.Mui-focused': {
    color: focusedColor,
  },
  '& .MuiInput-underline:after': {
    borderBottomColor: focusedColor,
  },
  '& .MuiInputBase-input': {
    fontSize: labelFontSize,
  },
}
```

like 
```
MuiFormControl: {
      styleOverrides: {
        root: {
          marginBottom: '1.2rem',
          ...rootBaseStyle
       },
```

**Review by mwpark2014 — APPROVED** [review]
Overall lgtm! Good work! Please acknowledge the feedback and then decide if you want to merge

File: client/src/components/App/App.css (hunk 1 above)
Comment (hmzakhalid) [reply client/src/components/App/App.css:32]: Yes, this file got auto formatted. there are no changes in this

File: client/src/components/Form/FormTheme.js (hunk 2 above)
Comment (hmzakhalid) [reply client/src/components/Form/FormTheme.js]: I didn't think of that, Thanks!
I've made the changes.

**Comment by hmzakhalid** [comment]
Thanks for the feedback!
I did the changes

**Comment by mwpark2014** [comment]
@zoobot I'm going to squash and merge this PR!

**Pull request merged**